cypress/support/a11y/checkAxe.js

```diff
@@ -6,9 +6,11 @@
  * Side Public License, v 1.
  */
 
-import { context, defaultAxeConfig } from './defaultAxeConfig';
+type Result = import('axe-core').Result;
```

Comment:
Now that we're not in a `.d.ts` file, you should be able to just import types normally

```suggestion
import { Result } from 'axe-core';
```

Actually, is there anything stopping us from making this a `.ts` file instead of `.js`?

Reply:
Those are my last minor comments, but they're definitely not blocking, so feel free to merge away!